client/screen.css

```diff
@@ -13,7 +17,7 @@ body {
 }
 
 .county-root .info {
-    display: flex;
+  #  display: flex;
```

Comment:
'tis, but it does have the effect of making this declaration invalid (except, I guess, on IE7, which I know was an accident).

The *actual* impact of this is to put "contest info" below "county info" on the main county dashboard page, rather than having "contest info" to the right, which is how it works with `display: flex` on.

I'm going to wrap this in a real CSS comment and put my own comment above it to explain why it's like that - that way we're faithful to their existing layout for the time being.

If you don't have any objections, that'll be the only other change I make before I merge this.